.ci/Jenkinsfile

```diff
@@ -53,6 +53,9 @@ pipeline {
               "^test/instrumentation/modules/"
             ]
             env.TAV_UPDATED = isGitRegionMatch(patterns: regexps)
+
+            // Skip all the stages except docs for PR's with asciidoc changes only
+            env.ONLY_DOCS = isGitRegionMatch(patterns: [ '.*\\.asciidoc' ], comparator: 'regexp', shouldMatchAll: true)
```

Comment:
We should include `.md` as well

Reply:
are those files hosted in the same folder?

Reply:
No, all the markdown files are currently in the root of the repo (where there's also one asciidoc file). The rest of the asciidoc files are all in the `docs` directory). I thought this pattern check was for all files no matter where in the repo they were located?

Reply:
Come to think of it, for markdown changes, there's no reason to run any CI checks at all (besides to check if the committer have signed the CLA). For asciidoc changes, we should still run the `elasticsearch/docs` step

Reply:
got it, I'll then use a pattern like `'.*\\.(asciidoc|md)'` then all the changes in the PR should be markdown or asciidoc files.